### PR TITLE
slider added

### PR DESCRIPTION
**Pull request by jeetbhalani15**
image

### CODE REVIEW

File: components/Slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.box{
+    background-color: white;
+    padding: 20px 20px 20px 30px;
+    border-radius: 5px;
+    box-shadow: 0px 0px 5px rgba(0, 0, 0, .1);
+    display: flex;
```
Comment (aniketsingh13) [inline components/Slider/slider.css:10]: you can use utility for this

File: components/Slider/slider.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    align-items: center;
+}
+ .box .slider{
+    height: 40px;
+    width: 300px;
+    display: flex;
+    align-items: center;
+    margin-right: 15px;
```
Comment (aniketsingh13) [inline components/Slider/slider.css:22]: here also

File: components/Slider/slider.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+}
+.box .slider input{
+    height: 10px;
+    width: 100%;
+    outline: none;
+    background: #f2f2f2;
+    border-radius: 25px;
+    box-shadow: inset 0px 0px 4px rgba(0, 0, 0, .2);
```
Comment (aniketsingh13) [inline components/Slider/slider.css:30]: make color utility for consistency

File: components/Slider/slider.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      <div class="value">100</div>
+    </div>
+    <script>
+        const slider = document.querySelector("input");
+        const value = document.querySelector(".value");
+        value.textContent = slider.value
+        slider.oninput = function(){
+            value.textContent = this.value;
+        }
+    </script>
```
Comment (aniketsingh13) [inline components/Slider/slider.html:24]: it is good practice to use js in seperate file

**Review by aniketsingh13 — APPROVED** [review]
other thing good to go. consider the changes

File: components/Slider/slider.html (hunk 4 above)
Comment (jeetbhalani15) [reply components/Slider/slider.html:24]: ok sure

**Review by Jay-Gadhiya — APPROVED** [review]

**Pull request merged**